knowledge-base/gauge-pointer-tooltip.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Tooltip for the Gauge Pointer
+description: How to show a Tooltip on hover of the Gauge Pointer?
+type: how-to
+page_title: Tooltip for the Gauge Pointer
+slug: gauge-kb-pointer-tooltip
+position: 
+tags: gauge,tooltip,pointer,arc,circular,linear,radial
+ticketid: 1535827
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Arc Gauge for Blazor,<br />
+                Circular Gauge for Blazor,<br />
+                Linear Gauge for Blazor,<br />
+                Radial Gauge for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+I want to show a Tooltip when hovering the pointer of the Radial Gauge. How to achieve it? 
```

Comment:
Try to build a logically consistent description without listing different questions.
```suggestion
How can I add a Tooltip to the Blazor Arc or Radial Gauge? I want to show a Tooltip when the user hovers over the pointer of the Radial Gauge or the Arc Gauge.
```